# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../cvs/getline-nameclash.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- cvs-1.11.22.1/lib/getline.h.orig	2010-03-26 22:56:08.000000000 +0100
+++ cvs-1.11.22.1/lib/getline.h	2010-03-26 22:57:03.000000000 +0100
@@ -12,7 +12,7 @@
 #define GETLINE_NO_LIMIT -1
 
 int
-  getline __PROTO ((char **_lineptr, size_t *_n, FILE *_stream));
+  cvs_getline __PROTO ((char **_lineptr, size_t *_n, FILE *_stream));
 int
   getline_safe __PROTO ((char **_lineptr, size_t *_n, FILE *_stream,
                          int limit));
--- cvs-1.11.22.1/lib/getline.c.orig	2010-03-26 22:56:19.000000000 +0100
+++ cvs-1.11.22.1/lib/getline.c	2010-03-26 22:57:23.000000000 +0100
@@ -155,7 +155,7 @@
 }
 
 int
-getline (lineptr, n, stream)
+cvs_getline (lineptr, n, stream)
      char **lineptr;
      size_t *n;
      FILE *stream;
